core/lib/via_btc_client/src/inscriber/mod.rs

```diff
@@ -611,54 +654,94 @@ impl Inscriber {
         Ok(res)
     }
 
+    #[instrument(skip(self, commit, reveal), target = "bitcoin_inscriber")]
     async fn broadcast_inscription(&self, commit: &FinalTx, reveal: &FinalTx) -> Result<bool> {
+        info!("Broadcasting inscription transactions");
         let commit_tx_hex = commit.tx.raw_hex().to_string();
         let reveal_tx_hex = reveal.tx.raw_hex().to_string();
 
         let mut commit_broadcasted = false;
         let mut reveal_broadcasted = false;
 
-        // broadcast commit tx with retry
-        for _ in 0..BROADCAST_RETRY_COUNT {
+        for attempt in 1..=BROADCAST_RETRY_COUNT {
+            debug!(
+                "Attempting to broadcast commit transaction (attempt {})",
+                attempt
+            );
             let res = self
                 .client
                 .broadcast_signed_transaction(&commit_tx_hex)
                 .await;
 
             if res.is_ok() {
                 commit_broadcasted = true;
+                info!("Commit transaction broadcasted successfully");
                 break;
+            } else {
+                warn!(
+                    "Failed to broadcast commit transaction (attempt {})",
+                    attempt
+                );
             }
         }
 
-        // broadcast reveal tx with retry
-        for _ in 0..BROADCAST_RETRY_COUNT {
+        for attempt in 1..=BROADCAST_RETRY_COUNT {
```

Comment:
We could probably remove the retries here, since the rpc_client will handle them?